fitlins/workflows/base.py

```diff
@@ -5,29 +5,33 @@
 
 
 def init_fitlins_wf(bids_dir, preproc_dir, out_dir, space,
-                    model=None, participants='.*',
+                    model=None, participants=None,
                     base_dir=None, name='fitlins_wf'):
     wf = pe.Workflow(name=name, base_dir=base_dir)
 
-    specs = ModelSpecLoader(bids_dirs=bids_dir)
+    specs = ModelSpecLoader(bids_dir=bids_dir)
     if model is not None:
         specs.inputs.model = model
 
     all_models = specs.run().outputs.model_spec
     if not all_models:
         raise RuntimeError("Unable to find or construct models")
 
+    selectors = {'subject': participants} if participants is not None else {}
+
     loader = pe.Node(
-        LoadLevel1BIDSModel(bids_dirs=[bids_dir, preproc_dir],
-                            selectors={'subject': participants}),
+        LoadLevel1BIDSModel(bids_dir=bids_dir,
+                            preproc_dir=preproc_dir,
```

Comment:
You can't set `preproc_dir=None`, which is the default. This will need to be:

```Python
loader = pe.Node(
    LoadLevel1BIDSModel(bids_dir=bids_dir, selectors=selectors),
    name='loader')
if preproc_dir is not None:
    loader.inputs.preproc_dir = preproc_dir
```

Reply:
Ah. Is there no way to make it nullable with Traits?

Reply:
You could define a trait like that, but I'd rather do type wizardry in Nipype than here.